plugin-networkmonitor/translations/networkmonitor_ro_RO.ts

```diff
@@ -58,7 +58,7 @@
     <message>
         <location filename="../lxqtnetworkmonitorconfiguration.ui" line="14"/>
         <source>Network Monitor settings</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Setări monitorizare rețea</translation>
```

Comment:
@jubalh Should I remove the *unfinished* type attribute after updating translations or will those attributes be updated in a later pass?